lesson7-questions.txt

```diff
@@ -87,19 +99,25 @@ You will see the SQL that is executed logged to the screen.  What is that SQL?
 There is a security exposure, a risk that a hacker could do damage.  What is that
 risk? What two things are done in your code to prevent the risk?
 
+unauthorized attributes could be changed or created
+adding a validation can prevent the risk
+checking the return value
+using a before_action you can make sure the user is authorized before the session is destroyed
```

Comment:
we're not destroying the session. though the session does help us know if the user is authorized to perform this action! 

I think you have the first one here -- how does the `before_action` prevent an unauthorized user from destroying someone else's post?

There's also another safeguard in accessing the post in the first place. Where would that be?